src/main/java/core/basesyntax/MyLinkedList.java

```diff
@@ -3,45 +3,181 @@
 import java.util.List;
 
 public class MyLinkedList<T> implements MyLinkedListInterface<T> {
+    private Node<T> first;
+    private Node<T> last;
+
     @Override
     public void add(T value) {
+        Node<T> newNode = new Node<>(value);
```

Comment:
```suggestion
        Node<T> newNode = new Node<>(last, value, null);
```
![image](https://github.com/mate-academy/jv-linked-list/assets/20326858/8c21da79-3c40-4625-a339-35568160d184)